docs/pages/lunatrace/overview/introduction.md

```diff
@@ -0,0 +1,25 @@
+---
+id: "introduction"
+title: "Introduction"
+sidebar_label: "Introduction"
+sidebar_position: 1
+---
+
+<!--
+  ~ Copyright by LunaSec (owned by Refinery Labs, Inc)
+  ~
+  ~ Licensed under the Creative Commons Attribution-ShareAlike 4.0 International
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License. You may obtain a copy of the License at
+  ~
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+## What is LunaTrace
+
+LunaTrace is a vulnerability scanner that tags and tracks inventories through to production deployment.  It combines static
+analysis at build time with runtime tracking and the capability to automatically patch some high-profile vulnerabilities.
```

Comment:
I'm sure we will flesh this out a little more in the future. I have some ideas about what to put here so maybe ill make a pass on it later

Reply:
Makes sense, I basically just am giving the README a place to live as we want to develop the doc in the new structure